apps/studio/src/constants/formBuilder.ts

```diff
@@ -3,6 +3,7 @@ export const JSON_FORMS_RANKING = {
   ArrayControl: 4,
   BooleanControl: 2,
   DropdownControl: 2,
+  ImageControl: 2,
```

Comment:
can i understand how this rank was chosen?

Reply:
So a higher ranking will be a higher priority. For the image field, since it's also a text field(textcontrol has ranking 1), I gave it the smallest number greater than 1 since we need to have a higher priority than the default text field.